credits.txt

```diff
@@ -118,32 +121,36 @@ Beta Testers
 Thank you to all the following
 people on GitHub who have
 contributed to Endless Sky:
+  10010101001
   7even
   abenkovskii
-  AdamKauffman
   akien-mga
   AlbertNewton
   AlexBassett
   alextd
   Alkallid
+  Amacita
   Amazinite
   AMDmi3
   anarcat
   Anarchist2
   AraCaputDraco
   Arachi-Lover
+  a-random-lemurian
   arkhne
+  Arrow2thekn33
   ashdnazg
   Aurelite
   barthalion
   beccabunny
+  benflodge
   BlackVegetable
   Bladewood
+  BlazingDiesel
   bobrobbow
   bojidar-bg
   Brick63
   ChamEV
-  colbygallup
```

Comment:
I can't find anything this user has contributed, but since you made the original list, maybe a Github issue? Either way, maybe keep this user. This more or less applies to everyone removed, are we sure they haven't contributed?

Reply:
Was added in #4962, fairly sure that was deliberate. Voting to keep it